app/helpers/application_helper.rb

```diff
@@ -157,6 +161,15 @@ def cache_key_heroku_slug(path)
     "#{path}-#{heroku_slug_commit}"
   end
 
+  def copyright_notice
+    start_year = ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"]
+    current_year = Time.current.year
```

Comment:
```suggestion
    current_year = Time.current.year.to_s
```

Here `start_year` will be a string as per the `Envfile`, while `current_year` will be an integer, so they'll never be equal:

<img width="709" alt="Screen Shot 2020-03-31 at 10 39 46" src="https://user-images.githubusercontent.com/47985/77984529-fcd05d80-733b-11ea-87bb-95c2a05dca0f.png">

This way we'll also consistently return strings from all early exits.